webapp/portlet/src/main/webapp/common/components/ExoActivityRichEditor.vue

```diff
@@ -52,6 +52,9 @@ export default {
         if (val === '') {
           this.initCKEditor();
         } else {
+          // reinitialize CKEditor instance
```

Comment:
This change is no more needed, please rzmove it from your PR

Reply:
okay